pkg/virtctl/vmexport/vmexport.go

```diff
@@ -857,3 +897,131 @@ func getExportSecretName(vmexportName string) string {
 func errExportAlreadyExists(err error) bool {
 	return strings.Contains(err.Error(), "VirtualMachineExport") && strings.Contains(err.Error(), "already exists")
 }
+
+// Port-forward functions
+
+// translateServicePortToTargetPort tranlates the specified port to be used with the service's pod
+func translateServicePortToTargetPort(localPort string, remotePort string, svc k8sv1.Service, pod k8sv1.Pod) ([]string, error) {
```

Comment:
I don't understand the purpose of this function, you can port forward to a service, you don't really care about the pod behind the service.

Reply:
This is part of the service-pod translation that's needed to do the port-forwarding. Kubectl does all of this under the hood, so I think it's best to replicate that behavior and avoid invoking kubectl commands from virtctl.